code/game/gamemodes/clock_cult/clock_scripture.dm

```diff
@@ -0,0 +1,1010 @@
+
+
+
+/datum/clockwork_scripture/create_object/tinkerers_daemon //Tinkerer's Daemon: Creates a shell that can be attached to a tinkerer's cache to grant it passive component creation.
+	name = "Tinkerer's Daemon"
+	desc = "Forms a daemon shell that can be attached to a tinkerer's cache to add new components at a healthy rate. It will only function if it is outnumbered by servants in a ratio of 5:1."
+	invocations = list("Pbaf'gehpg, Ratvar cnegf...", "...lrg ubyq terngarff!")
+	channel_time = 40
+	required_components = list("guvax_capacitor" = 1, "replicant_alloy" = 3, "hierophant_ansible" = 1)
+	object_path = /obj/item/clockwork/daemon_shell
+	creator_message = "<span class='brass'>You form a daemon shell. Attach it to a tinkerer's cache to increase its rate of production.</span>"
+	usage_tip = "Vital to increasing component generation."
+	tier = SCRIPTURE_APPLICATION
+
+//////////////
+// REVENANT //
+//////////////
+//Revenant scriptures are different than any others. They are all very powerful, but also very costly and have drawbacks. This might be a very long invocation time or a very high component cost.
+
+/datum/clockwork_scripture/invoke_nezbere //Invoke Nezbere, the Brass Eidolon: Invokes Nezbere, bolstering the strength of many clockwork items for one minute.
+	name = "Invoke Nezbere, the Brass Eidolon"
+	desc = "Taps the limitless power of Nezbere, one of Ratvar's four generals. The restless toil of the Eidolon will empower a wide variety of clockwork apparatus for a full minute - notably, \
+	clockwork proselytizers will cost no replicant alloy to use."
+	invocations = list("V pnyy hcba lbh, Nezbere!!", "Yrg lbhe znpuv'angvbaf ervta ba guvf zvfrenoyr fgng'vba!!", "Yrg lbhe cbjre sybj gueb-htu gur gbbyf bs lbhe znfgre!!")
+	channel_time = 150
+	required_components = list("belligerent_eye" = 3, "vanguard_cogwheel" = 3, "guvax_capacitor" = 3, "replicant_alloy" = 6)
+	usage_tip = "Ocular wardens will become empowered, clockwork proselytizers will require no alloy, tinkerer's daemons will produce twice as quickly, and mending motors will require no alloy."
+	tier = SCRIPTURE_REVENANT
+
+/datum/clockwork_scripture/invoke_nezbere/check_special_requirements()
+	if(ratvar_awakens)
+		invoker << "<span class='heavy_brass'>\"Bhe znfgre vf urer nyernql. Lbh qb abg erdhver zl uryc, sevraq.\"</span>\n\
+		<span class='warning'>Nezbere will not grant his power while Ratvar's dwarfs his own!</span>"
+		return 0
+	if(clockwork_generals_invoked["nezbere"])
+		invoker << "<span class='heavy_brass'>\"Abg whfg lrg, sevraq. Cngvrapr vf n iveghr.\"</span>\n\
+		<span class='warning'>Nezbere has already been invoked recently! You must wait several minutes before calling upon the Brass Eidolon.</span>"
+		return 0
+	return 1
+
+/datum/clockwork_scripture/invoke_nezbere/scripture_effects()
+	world << "<font color='#BE8700' size=5><b>\"V urrq lbhe pnyy, punz'cvbaf. Znl lbhe negvs-npgf oevat ehva hcba gur urnguraf gung bccbfr bhe znfgre!\"</b></font>\n\
```

Comment:
People keep going "what the fuck was that was that ratvar" when this is used
also, for each of these revenant scriptures you need to check if the general was invoked, so people can't, like, spam one of these scriptures to get super-whatevers.